Informatics I/011 Word Game and Comprehensions/Task 2 Hex-Code Generator/Patertuck Solution/script.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+from random import choice
+
+class GameRunner(object):
+
+    def __init__(self):
+        self.rows = 17
+        self.columns = 2
+
+    def generate_hex_codes(self):
+        return [("0x" + "".join([choice("0123456789ABCDEF") for _ in range(4)]))for _ in range (self.columns * self.rows)]
```

Comment:
very nice! (minor thing: missing space before "for" and additional after "range")

Reply:
ah yes, is this a change that is worth me reopening the branch and fixing it?